packages/cli/tasks/interact.js

```diff
@@ -18,8 +23,15 @@ task(TASK_INTERACT, 'Interacts with a given modular system deployment')
     const { debug } = taskArguments;
 
     logger.debugging = debug;
+    const escItem = '↩ BACK';
```

Comment:
If this is used by subtasks, we should probably abstract it to a constants file.